docs/publishers/config.rst

```diff
@@ -407,11 +407,9 @@ loads.
         focus: {
           user: {
             // required
-            username: "foobar1234",
-
+            userid: "acct:foobar1234@authority",
```

Comment:
I note this is a breaking change from the previous API. The "focus" mode is marked as beta so that's OK, but it is possible that some third-party might run into this.

Reply:
Okay, I won't make this change this round, make a separate ticket to clean this up later.